examples/internal/proto/examplepb/unannotated_echo_service.swagger.json

```diff
@@ -449,10 +449,7 @@
           "type": "string",
           "format": "int64",
           "default": "42",
-          "description": "Int value field",
-          "required": [
-            "num"
-          ]
```

Comment:
it's already there [https://github.com/grpc-ecosystem/grpc-gateway/blob/adad28a213663eed845865fb288fe1a83d7849a8/examples/internal/proto/examplepb/unannotated_echo_service.swagger.json#L483](url) here,if you remember the issue i am trying to solve is to avoid override fields in required section